pkgs/development/ocaml-modules/mm/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildDunePackage, fetchFromGitHub, dune-configurator
+, alsa, ao, mad, pulseaudio, theora
+}:
+
+buildDunePackage rec {
+  pname = "mm";
+  version = "0.8.1";
+
+  minimalOCamlVersion = "4.12";
+
+  src = fetchFromGitHub {
+    owner = "savonet";
+    repo = "ocaml-mm";
+    rev = "v${version}";
+    sha256 = "sha256-7ozt+OgKNxMnjl2R+/ce27ZyL+T6BShvnnFE5BasJC4=";
+  };
+
+  buildInputs = [ dune-configurator ];
+  propagatedBuildInputs = [ alsa ao mad pulseaudio theora ]; # ocamlsdl is blocked in nixpkgs from building for ocaml >= 4.06
```

Comment:
I don’t understand the comment. What does it mean?

Reply:
ocamlsdl is an optional dependency for mm, but the minimalOcamlversion in mm is above the maximum version allowed for  ocamlsdl in nixpkgs. See https://github.com/NixOS/nixpkgs/blob/a43da8cc8b1f2ed2849f14b8f8f18f813883b7e3/pkgs/development/ocaml-modules/ocamlsdl/default.nix#L7-L9